util/cron/test-linux64-python35.bash

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Test default configuration with `make check`, on linux64, with python 3.5
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+source $CWD/common.bash
```

Comment:
Could we have `common-python.bash` source `common.bash`? This would make it similar to `common-gasnet.bash`. Arguably we could go in the opposite direction for `common-gasnet.bash` if you're opposed to it.

Reply:
Good point. If it's up to me, I would try to avoid `common`s sourcing each other. My rationale for that is to avoid unexpected environment variables to be set for some jobs. And forcing all job scripts to `source` what they need explicitly can be a way to avoid that.


Maybe there is some thing in `common-gasnet` that depends on `common` though.

Reply:
It could be that `common-gasnet.bash` needs something from `common.bash` and I just didn't realize it. Let's leave things as is.